haystack/utils/export_utils.py

```diff
@@ -48,15 +48,16 @@ def print_answers(results: dict, details: str = "all", max_text_len: Optional[in
     elif details == "all":  
         filtered_answers = answers
     else:
-        logging.warn(f"print_answers received details='{details}', which was not understood. "
-                     "Valid values are 'minimum', 'medium', and 'all'. Using 'all'.")
+        valid_values = ", ".join(fields_to_keep_by_level.keys()) + " and 'all'"
+        logging.warn(f"print_answers received details='{details}', which was not understood. ")
+        logging.warn(f"Valid values are {valid_values}. Using 'all'.")
         filtered_answers = answers
 
     # Shorten long text fields
-    if max_text_len is not None:
+    if max_text_len:
```

Comment:
We generally prefer to be explicit about the difference between `None` and `0`, especially where the variable is an int.